From a6b0fde8f776d28988a9ab4e5bf09f78fa858744 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Sun, 10 Mar 2013 13:31:02 +0100
Subject: [PATCH 3/3] Revert "crypt, dmraid, mdraid: use
 for_each_host_dev_and_slaves_all()"

This reverts commit cf24ac8c19b803e6f5c0576f2d7e78187331cf02.
---
 modules.d/90crypt/module-setup.sh  | 2 +-
 modules.d/90dmraid/module-setup.sh | 2 +-
 modules.d/90mdraid/module-setup.sh | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/90crypt/module-setup.sh b/modules.d/90crypt/module-setup.sh
index 2b4456b..93b996c 100755
--- a/modules.d/90crypt/module-setup.sh
+++ b/modules.d/90crypt/module-setup.sh
@@ -25,7 +25,7 @@ check() {
     }
 
     [[ $hostonly ]] || [[ $mount_needs ]] && {
-        for_each_host_dev_and_slaves_all check_crypt || return 1
+        for_each_host_dev_fs check_crypt || return 1
     }
 
     return 0
diff --git a/modules.d/90dmraid/module-setup.sh b/modules.d/90dmraid/module-setup.sh
index c315961..eb949dc 100755
--- a/modules.d/90dmraid/module-setup.sh
+++ b/modules.d/90dmraid/module-setup.sh
@@ -38,7 +38,7 @@ check() {
     }
 
     [[ $hostonly ]] || [[ $mount_needs ]] && {
-        for_each_host_dev_and_slaves_all check_dmraid || return 1
+        for_each_host_dev_fs check_dmraid || return 1
     }
 
     return 0
diff --git a/modules.d/90mdraid/module-setup.sh b/modules.d/90mdraid/module-setup.sh
index 0efffbb..a2b9ac2 100755
--- a/modules.d/90mdraid/module-setup.sh
+++ b/modules.d/90mdraid/module-setup.sh
@@ -27,7 +27,7 @@ check() {
     }
 
     [[ $hostonly ]] || [[ $mount_needs ]] && {
-        for_each_host_dev_and_slaves_all check_mdraid || return 1
+        for_each_host_dev_fs check_mdraid || return 1
     }
 
     return 0
-- 
1.8.1.4

